src/Command/DumpEncryptedFieldsMapCommand.php

```diff
@@ -63,24 +61,27 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $dumper = new Dumper();
 
         foreach ($this->documentManagers as $name => $documentManager) {
-            $generator          = new EncryptedFieldsMapGenerator($documentManager->getMetadataFactory());
-            $encryptedFieldsMap = $generator->getEncryptedFieldsMap();
+            assert($documentManager instanceof DocumentManager);
+
+            $encryptedFieldsMap = [];
+            foreach ($documentManager->getMetadataFactory()->getAllMetadata() as $metadata) {
+                $database               =  $documentManager->getDocumentDatabase($metadata->getName());
+                $collectionInfoIterator = $database->listCollections(['filter' => ['name' => $metadata->getCollection()]]);
```

Comment:
This is inspired by the [`get_encrypted_fields_from_server` function](https://github.com/mongodb/mongo-php-library/blob/0058bc008028d91f5f9e032e80daa094b31ef9a9/src/functions.php#L201).